modules/testbase.lua

```diff
@@ -128,6 +128,29 @@ function control.runNextCase()
     xmlReporter.AddCase(Test.current_case_name)
     atf_logger.LOGTestCaseStart(Test.current_case_name)
     testcase(Test)
+
+    local function wait(pConnection)
+      local timeout = config.unexpectedEventTimeout
+      local event = events.Event()
+      event.matches = function(e1, e2) return e1 == e2 end
```

Comment:
@dboltovskyi Please use more descriptive parameter names here.

Reply:
@aderiabin Done in 0913260